packages/web-app-files/src/helpers/contextualHelpers.js

```diff
@@ -46,6 +46,9 @@ export const shareViaLinkHelp = {
     )
   ]
 }
+export const shareViaIndirectLinkHelp = {
+  text: $gettext('This file is shared, because one of the folders it is in is shared via link.')
```

Comment:
What about something like `This file is shared, because one of the parent folders is shared via link.`? Is parent folder too technical?

Reply:
@tbsbdr suggested so IIRC, but I'm happy with either (or another suggestion)!

Reply:
@kulmann yes, I guess "parent folder" is quite technical, thats why I suggested the "folders it is in" text.

Reply:
@mmattel do you have an opinion if we should write "parent" or "folders it is in"?
![screenie_Brave Browser_2022-06-20_at_14-14-21@2x](https://user-images.githubusercontent.com/26610733/174599654-d7f7d2c5-c88a-4660-9520-7d2eaee87c85.png)